config/karma.config.js

```diff
@@ -3,11 +3,12 @@
 
 import webpackConfig from './webpack.config';
 
-var coverageReporters = [{
+const coverageReporters = [{
   type: 'text-summary',
 }];
 
-var newWebpackConfig = Object.assign({}, webpackConfig, {
+// eslint-disable-next-line prefer-const
+let newWebpackConfig = Object.assign({}, webpackConfig, {
```

Comment:
Thanks for the links and discussion points. I definitely feel like I got the wrong end of the stick re: `const` since I'd taken to imply 'this won't be changed' more than it it being 'this won't be rebound' despite knowing that a const object could be mutated without errors. Maybe eslint could use something like "this isn't rebound" in the error message for prefer-const. 

I'll fix up this file for now - and we should all have a think about what a sane default going forward would be.